src/test/java/org/reaktivity/reaktor/ControllerIT.java

```diff
@@ -83,7 +83,7 @@ public void shouldRouteAsServer() throws Exception
     @Specification({
         "${control}/route/server/controller"
     })
-    @ScriptProperty("authorization [0x01 0x00 0x00 0x00 0x00 0x00 0x00 0x00]")
+    @ScriptProperty("routeAuthorization [0x01 0x00 0x00 0x00 0x00 0x00 0x00 0x00]")
```

Comment:
Should this be `long` literal hex instead of `byte[]` ?

Reply:
fixed